pkg/sql/show.go

```diff
@@ -414,6 +414,20 @@ func (p *planner) showCreateTable(tn parser.Name, desc *sqlbase.TableDescriptor)
 	return buf.String(), nil
 }
 
+func makeIndexColNames(d sqlbase.IndexDescriptor) string {
+	var buf bytes.Buffer
+	for i, name := range d.ColumnNames {
+		if i > 0 {
+			buf.WriteString(", ")
+		}
+		buf.WriteString(parser.Name(name).String())
+		if dir := d.ColumnDirections[i]; dir != 0 {
+			buf.WriteString(" " + dir.String())
```

Comment:
Either `fmt.Fprintf(" %s", dir)` or `buf.WriteByte(' '); buf.WriteString(dir.String())`, the string concatenation is not warranted.